.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - '**'
```

Comment:
What does this do? 

Can't you just leave it out as it is the default I guess?

Reply:
The default for [push](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push) is:
> Runs your workflow when you push a commit or tag.

As we have a separate workflow for tags, I added the `branches` section. It accepts standard glob patterns, which this is. But you might be right, it could work with `branches` but without any list entries (but it's essentially equivalent).

Reply:
Just noticed that it's not the case, GH Action isn't triggered for this branch anymore with `branches` but no glob:
 
![image](https://user-images.githubusercontent.com/4820162/205304494-9c1a0884-93bd-4148-a702-a3350c9bca48.png)

Reply:
Yes, but then it is also triggered for tags. As tags are covered by a separate workflow, I'd suggest leaving the `branches` in. Or why do you see this as such an issue?

Reply:
Discussed offline, fine to keep as is.